tests/grammar_test.py

```diff
@@ -3473,7 +3473,7 @@ def test_session_declaration(string: str, expected: Dict[str, str]) -> None:
                 "_kind": "TypeDecl",
                 "definition": {
                     "_kind": "MessageTypeDef",
-                    "checksums": None,
+                    "aspects": [],
```

Comment:
Message aspects are not tested at all at the moment. Please add a test (e.g., by adding a test case to `test_type_declaration`).

Reply:
done

Reply:
done